src/main/java/model/FruitTransaction.java

```diff
@@ -0,0 +1,77 @@
+package model;
+
+public class FruitTransaction {
+    private Operation operation;
+    private String fruit;
+    private int quantity;
+
+    public FruitTransaction(String letter, String fruit, int quantity) {
+        this.operation = getOperationByLetter(letter);
+        this.fruit = fruit;
+        this.quantity = quantity;
+    }
+
+    private Operation getOperationByLetter(String letter) {
+        switch (letter) {
+            case "s":
+                return Operation.SUPPLY;
+            case "r":
+                return Operation.RETURN;
+            case "p":
+                return Operation.PURCHASE;
+            case "b":
+            default:
+                return Operation.BALANCE;
+        }
+    }
```

Comment:
let's move this method inside the enum and let's not use the switch case there 

look at the method fromString on screenshot
<img width="675" alt="Screenshot 2022-02-18 at 10 48 23" src="https://user-images.githubusercontent.com/33026471/154649301-d60410df-6344-4e08-8d14-207694aaafb0.png">